sdcm/data_operations/data_operation.py

```diff
@@ -0,0 +1,194 @@
+        with self.schema_manage.cluster.cql_connection_patient(self.schema_manage.cluster.nodes[0],
+                                                               keyspace=self.schema_manage.keyspace_name) as session:
+            for row in where_condition_to_delete:
+                self.query_one_partition(where_condition_to_delete=row, session=session)
+
+    def count_rows(self, entity):
+        try:
+            with self.schema_manage.tester.db_cluster.cql_connection_patient(self.schema_manage.tester.db_cluster.nodes[0],
+                                                                             keyspace=self.schema_manage.keyspace_name) as session:
+                query = f"select count(*) cnt from {entity}"
+                self.log.debug("Run select query: %s", query)
+                result = session.execute(query, timeout=300)
+                self.log.debug(
+                    "Rows in the %s.%s: %s", self.schema_manage.keyspace_name, self.schema_manage.table_name, result.one().cnt)
+                return result.one().cnt
+        except Exception as err:  # pylint: disable=broad-except
+            self.log.error("Failed query: %s. Error: %s", query, err)
+            return None
+
+
+# pylint: disable=too-many-instance-attributes
+class DataOperationThread:
+    # TODO: add description
+    """
+    Run thread with data manipulations during entire test. Sleep between operations.
+    For now one operation is supported: delete full partition
+    """
+    DEFAULT_INTERVAL_BETWEEN_OPERATIONS_SEC = 600
+
+    def __init__(self, tester, thread_name: str):
+        self.tester = tester
+        self.schema_manage = SchemaManage(tester=self.tester)
+        self.data_operations = DataOperations(schema_manage=self.schema_manage)
+        # TODO: maybe changed, expanded with new parameters
+        # parameter "data_operation_ks_class" defined as:
+        #  - {'replication_factor': 3, 'replication_strategy': 'SimpleStrategy'}
+        #  OR
+        #  - {'replication_factor': ['eu-westscylla_node_west': 3, 'us-west-2scylla_node_west': 2],
+        #     'replication_strategy': 'NetworkTopologyStrategy}
+        ks_class = self.tester.params.get("data_operation_ks_class")
+        self.schema_manage.create_default_keyspace_query(replication_factor=ks_class["replication_factor"],
+                                                         replication_strategy=ks_class["replication_strategy"])
+        self._thread = threading.Thread(daemon=True, name=f"{self.__class__.__name__}_{thread_name}", target=self.run)
+        self.termination_event = threading.Event()
+        self.duration = self.tester.params.get('test_duration')
+        # TODO: do we want to config it in test yaml?
+        self.interval = self.DEFAULT_INTERVAL_BETWEEN_OPERATIONS_SEC
+        self.log = logging.getLogger(self.__class__.__name__)
+
+    def run(self):
+        end_time = time.time() + self.duration
+        self.schema_manage.run_user_profile()
```

Comment:
no sure I like the idea of running stress command here, and without checking it's results
can we start with using the main oad stress command ?, and enable it only on a specific case ?